test/e2e/kubelet_stats.go

```diff
@@ -155,14 +155,35 @@ func getContainerInfo(c *client.Client, nodeName string, req *server.StatsReques
 	if err != nil {
 		return nil, err
 	}
-	data, err := c.Post().
-		Prefix("proxy").
-		Resource("nodes").
-		Name(fmt.Sprintf("%v:%v", nodeName, ports.KubeletPort)).
-		Suffix("stats/container").
-		SetHeader("Content-Type", "application/json").
-		Body(reqBody).
-		Do().Raw()
+	subResourceProxyAvailable, err := serverVersionGTE(subResourceProxyVersion, c)
```

Comment:
`subResourceProxyVersion` is for `pods/proxy`, and is set to 1.1.0. That should be renamed to `subResourcePodProxyVersion` or something, and a new var added for nodes and services subresource proxy, since those will be in 1.2.0